scripts/gdb-pretty-printers.py

```diff
@@ -0,0 +1,151 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import gdb
+
+
+if not 'ThrustVectorPrinter' in locals():
```

Comment:
`not in` is the preferred spelling of this:
```suggestion
if 'ThrustVectorPrinter' not in locals():
```